django_feature_flip/helpers.py

```diff
@@ -19,4 +19,23 @@ def __getattr__(self, name):
         return getattr(self.instance, name)
 
     def enabled(self, feature_name, actor=None):
-        return Feature.objects.get(name=feature_name).enabled(actor)
+        feature = Feature.objects.get(name=feature_name)
```

Comment:
This will raise a `DoesNotExist` error if the `Feature` does not exist. Do we want that or should we catch the error and show some kind of message?